---
layout: archive
title: "Resume"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

&nbsp;

Work experience
======
* **October 2023 - Present:** Bioinformatician at ASRT Inc (CDC contractor)
  Enteric Diseases Laboratory Branch (EDLB), Centers for Disease Control and Prevention
  * Support Development and Operations activities of the PulseNet USA molecular surveillance network by addressing user issues, deploying scripts, modifying and updating tools.
  * Perform validation and benchmarking studies of Oxford Nanopore Technologies (ONT) sequencing assemblies against Illumina assemblies using core genome MLST (cgMLST) analysis using outbreak datasets and certification strains, to assess accuracy, reproducibility and performance.
  *	Execute comparative genomics and phylogenetic analyses on high-throughput datasets using cgMLST tools, to assess performance of various open-source cgMLST tools.
  *	Utilize High Performance Computing (HPC) environments daily for large-scale sequence processing, data management, and workflow optimization.
  *	Apply Python (matplotlib, scikit-learn) and R (ggplot2) for statistical modelling, regression analysis, and data visualization in validation studies.
  *	Monitor EDLB’s sequence data QA/QC system and investigate pipeline failures, respond to user requests, including transferring data and files when needed.
  *	Collaborate cross-functionally with bioinformaticians, scientists and epidemiologists to perform validation studies using cgMLST analysis for large-scale outbreak datasets.
&nbsp;

&nbsp;

* **January 2023 - September 2023:** ORISE Fellow (Bioinformatics, CDC)
  Surveillance, Information Management, and Statistics Office
  *	Evaluate and adapt existing bioinformatics tutorials to address the training needs of the division (DFWED).
  *	Serve as a technical resource to adapt existing resources and develop new training materials to impart bioinformatics knowledge.
  *	Developed evaluation metrics to assess the effectiveness of the bioinformatics trainings and tutorials. Modified future trainings based on the feedback received.
  *	Conducted training sessions for Waterborne Disease Prevention Branch (WDPB) with presentations on bioinformatics topics (e.g., Read mapping, Performing sequence quality control, De novo genome assembly, BLAST, etc.).
  *	Created slide decks and follow-along code documents to support the training materials.
  *	Collaborated with other senior bioinformaticians to maintain knowledge of current and widely used bioinformatics tools and algorithms within the organization.
  *	Tested multiple allele callers suitable for foodborne pathogens as part of an on-going project in EDLB. 
&nbsp;

&nbsp;

* **January 2022 - December 2022:** Data Analyst (Cooper lab)
  Department of Bioinformatics, University of North Carolina at Charlotte
  * Contributed to a project to identify structural variants and Single Nucleotide Polymorphisms (SNPs) that can be used for mapping studies. Investigated structural variants associated with Transposon Insertion-site Polymorphism (TIPs) to gain insights into the evolution of a plant genome (Sorghum).
  *	Assisted with the de novo assembly, scaffolding, and alignment of 11 new sorghum genomes.
  *	Performed high-throughput comparative genomic analysis of gene content, repetitive element abundance, and structural variation.
  *	Identified TIPs near or within genes that have the potential to affect plant phenotype.
  *	Wrote scripts in Bash, Python, and Perl to help implement and optimize genome analysis pipelines on additional data sets.
  *	Tested multiple methods for genome content analysis and optimized parameters for different data sets.
&nbsp;

&nbsp;

* **August 2021 - December 2021:** Research Assistant (Cooper lab)
  Department of Bioinformatics, University of North Carolina at Charlotte 
  * Contributed to develop a high-throughput pipeline for Transposable element Insertion Polymorphism (TIP) in wild and domesticated Sorghum.
  *	Wrote bash scripts to implement repeat identification and annotation pipelines on the HPC cluster.
&nbsp;

&nbsp;

* **August 2021 - December 2021:** Graduate Teaching Assistant (Dr. Roger's class on Sequence Analysis)
  Department of Bioinformatics, University of North Carolina at Charlotte
  *	Provided support for undergraduate lab sessions focused on genome mapping, variant calling, and sequence alignment.
  *	Guided students through hands-on bioinformatics exercises, helping them troubleshoot issues and understand core concepts.
  *	Graded assignments, quizzes, and exams, ensuring timely and constructive feedback.
  *	Led class discussions and managed lectures in the instructor’s absence.
  *	Held regular office hours to assist students with lab work and course material.
&nbsp;

&nbsp;

&nbsp;

Education
======
* M.S. in Bioinformatics, UNC Charlotte, 2022
* B.S. in Life Sciences and Biochemistry, St. Xavier's College Mumbai, 2015
&nbsp;

&nbsp;

&nbsp;

Skills
======

*	**Programming & Scripting:** Python, R, Bash
*	**Bioinformatics & Computational Tools:** NCBI, BLAST, sequence alignment tools, variant callers, MLST callers, genome assemblers, SAMtools, BEDtools
*	**Pipeline Development & Data Analysis:** Experience in developing and maintaining bioinformatics pipelines for WGS, Allele Calling, Genome assembly
*	**Reproducibility & Version Control:** Git, GitHub, GitLab, Conda, Singularity
*	**Statistical & Data Analysis:** Biostatistics, experimental design, data visualization, RStudio, Jupyter Notebooks, R Markdown
*	**High-Performance Computing (HPC):** Cluster computing, job scheduling, parallel processing
*	**Scientific Communication:** Experience in writing publications, preparing reports, and delivering bioinformatics training sessions



<!-- Publications
======
  <ul>{% for post in site.publications reversed %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul> -->
  
<!-- Talks
======
  <ul>{% for post in site.talks reversed %}
    {% include archive-single-talk-cv.html  %}
  {% endfor %}</ul>
  
Teaching
======
  <ul>{% for post in site.teaching reversed %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
Service and leadership
======
* Currently signed in to 43 different slack teams -->
